test/functional/configuration/cucumber/features/conf-service.feature

```diff
@@ -467,6 +475,14 @@ Feature: Road2 service configuration
     Then the configuration analysis should give an exit code 1
     Then the server log should contain "Mauvaise configuration: Champ 'application:sources:directories' manquant !"
 
+  Scenario: [service.json] (sources.directories sans aucune sources)
+    Given a valid configuration 
+    And an empty directory "empty_sources"
+    And with parameter "empty_sources" for attribute "application.sources.directories.[0]" in service configuration
```

Comment:
For tomorrow : I don't understand why this test passes... When I run it, it passes. But when I change my local conf, it doesn't pass. And that's the right way because some resources point to some sources which don't exist. 

This test should be : 
```Scenario: [service.json] (resources.directories sans aucune resource)
    Given a valid configuration 
    And an empty directory "empty_resource"
    And with parameter "empty_resource" for attribute "application.resources.directories.[0]" in service configuration
    And an empty directory "empty_sources"
    And with parameter "empty_sources" for attribute "application.sources.directories.[0]" in service configuration
    When I test the configuration
    Then the configuration analysis should give an exit code 0
    Then the server log should contain "Le dossier des resources est vide"
```

And another test could be : 
```
Scenario: [service.json] (sources.directories sans aucune source)
    Given a valid configuration 
    And an empty directory "empty_source"
    And with parameter "empty_source" for attribute "application.sources.directories.[0]" in service configuration
    When I test the configuration
    Then the configuration analysis should give an exit code 1
    Then the server log should contain "La ressource contient une source non disponible"
```

Reply:
Done inside the last commit